checkov/common/output/common.py

```diff
@@ -15,3 +15,32 @@ class ImageDetails(SCADetails):
     image_id: str = ''
     name: str | None = ''
     related_resource_id: str | None = ''
+
+
+def is_raw_formatted(licenses: str) -> bool:
+    return licenses.__contains__('","')
+
+
+def format_licenses_to_string(licenses_lst: list[str]) -> str:
+    print('format_licenses_to_string - start', licenses_lst)
```

Comment:
```suggestion
    logging.debug('format licenses to string', licenses_lst)
```
if you really want to log it, then please use `logging`

Reply:
I removed it